src/PKSim.Core/PKSim.Core.csproj

```diff
@@ -204,6 +204,7 @@
     <Compile Include="Model\RunParameterIdentificationQualificationStep.cs" />
     <Compile Include="Model\RunSimulationQualificationStep.cs" />
     <Compile Include="Services\IQualificationStepRunner.cs" />
+    <Compile Include="Services\MoleculeParameterTask.cs" />
```

Comment:
Added this new guy that is called from the previous Task. The reason is that the MoleculeExpressionTask does too much and more particularily, requires presentation. This needs to be split for Matlab to work properly

This is in general something we should try to make better. Split presentation tasks and core tasks. Not always possible or sometimes hard but required if we want to have a pure CORE implementation that does not rely on presentation at all